front-end/src/app/(home)/mode/page.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import Link from "next/link";
+import { Grid, Button, Typography, Box } from "@mui/material";
+import { create } from "@mui/material/styles/createTransitions";
+
+const mode_selection = () => {
+  return (
+    <Box
+      sx={{
+        height: "100vh",
+        backgroundColor: "black",
```

Comment:
Don't need these many styles. The dark theme has configured it will be by default.

Reply:
Thanks! Already fixed